vta/src/tsim/tsim_driver.cc

```diff
@@ -143,27 +133,15 @@ class Device {
   }
 
   void Launch(vta_phy_addr_t insn_phy_addr,
-              vta_phy_addr_t uop_phy_addr,
-              vta_phy_addr_t inp_phy_addr,
-              vta_phy_addr_t wgt_phy_addr,
-              vta_phy_addr_t acc_phy_addr,
-              vta_phy_addr_t out_phy_addr,
               uint32_t insn_count,
               uint32_t wait_cycles) {
-    dpi_->WriteReg(0x04, 0);
     dpi_->WriteReg(0x08, insn_count);
     dpi_->WriteReg(0x0c, insn_phy_addr);
-    dpi_->WriteReg(0x10, insn_phy_addr >> 32);
+    dpi_->WriteReg(0x10, 0);
```

Comment:
How about when we have VTA64 flag?

Reply:
`USE_VTA64` will require a little bit more thought in terms of hardware. I think for now we will try to match the HLS implementation as much as possible, so we can target current boards and then move forward from there. From what I see DE10 also support 32-bit addresses.

Reply:
Thanks for your explanation, I would agree with you on this.